tdrs-backend/tdpservice/email/tasks.py

```diff
@@ -16,10 +16,35 @@
 from tdpservice.email.email import automated_email, log
 from tdpservice.email.email_enums import EmailType
 from tdpservice.parsers.util import calendar_to_fiscal
+from tdpservice.email.helpers.account_deactivation_warning import send_deactivation_email
 
 
 logger = logging.getLogger(__name__)
 
+@shared_task
+def deactivate_users():
+    """Deactivate users that have not logged in in the last 180 days."""
+    users_to_deactivate = User.objects.filter(
+        last_login__lte=datetime.now(tz=timezone.utc) - timedelta(days=180),
+        account_approval_status=AccountApprovalStatusChoices.APPROVED,
+    )
+
+    for user in users_to_deactivate:
+        user.account_approval_status = AccountApprovalStatusChoices.DEACTIVATED
+        user.groups.clear()
```

Comment:
Why wouldn't we want to maintain the user's info intact? I don't see it in the ACs.

Reply:
Looks like it is the first AC

Reply:
the first AC reads: "the permission group should be de-selected for the user"